src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: fit-content;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star.svg");
+  margin-right: 4px;
+  border-radius: 0.5px;
```

Comment:
Why do you need this rule?
And do not use not integer values in `px`

Reply:
![Screenshot 2022-12-21 at 07 40 51](https://user-images.githubusercontent.com/22715618/208830305-4b089a1a-80f1-4fb9-beef-2a68df837f81.png)
I've seen the border in Figma mockup, ok I'll remove it